tests/scripts/fetch_target_logs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
+
+KUBE_CONFIG="/tmp/kubeconfig-${CLUSTER_NAME}.yaml"
+DIR_NAME="/tmp/target_cluster_logs"
+NAMESPACES="$(kubectl --kubeconfig="${KUBE_CONFIG}" get namespace -o json | jq -r '.items[].metadata.name')"
+mkdir -p "${DIR_NAME}"
+for NAMESPACE in ${NAMESPACES}
+do
+  mkdir -p "${DIR_NAME}/${NAMESPACE}"
+# Fetch logs from target cluster according to the pods name
+  PODS="$(kubectl --kubeconfig="${KUBE_CONFIG}" get pods -n "${NAMESPACE}" -o json | jq -r '.items[].metadata.name')"
+  for POD in ${PODS}
+  do
+    mkdir -p "${DIR_NAME}/${NAMESPACE}/${POD}"
+    CONTAINERS="$(kubectl --kubeconfig="${KUBE_CONFIG}" get pods -n "${NAMESPACE}" "${POD}" -o json | jq -r '.spec.containers[].name')"
+    for CONTAINER in ${CONTAINERS}
+    do
+      mkdir -p "${DIR_NAME}/${NAMESPACE}/${POD}/${CONTAINER}"
```

Comment:
`${DIR_NAME}/${NAMESPACE}/${POD}/${CONTAINER}"` is repeated few times in this loop. It would make sense to assign to `LOG_DIR` or such and use that. Makes entire loop much shorter and more readable.